ci-operator/step-registry/assisted/common/post/assisted-common-post-commands.sh

```diff
@@ -19,4 +19,4 @@ source /root/assisted-post-install.sh
 EOF
 
 echo "### Copying kubeconfig files"
-scp -F ${SHARED_DIR}/ssh_config -r root@ci_machine:/home/assisted/build/kubeconfig/* ${SHARED_DIR}
+scp -F "${SHARED_DIR}/ssh_config" -r "root@ci_machine:/home/assisted/build/kubeconfig/*" "${SHARED_DIR}" || true
```

Comment:
which scenario hits it?

Reply:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29138/rehearse-29138-pull-ci-openshift-assisted-test-infra-release-ocm-2.6-e2e-metal-assisted-ha-kube-api/1562718623368220672

Reply:
It's probably a bug in test-infra not being able to download the kubeconfigs, because those clusters are being installed pretty well
I wouldn't worry about it in the scope of this PR, but we can create a task to fix it